docroot/themes/humsci/humsci_basic/patterns/date-stacked-horizontal-card/date-stacked-horizontal-card.ui_patterns.yml

```diff
@@ -0,0 +1,47 @@
+date_stacked_horizontal_card:
```

Comment:
I've talked with H&S about this, all of these patterns don't have associated libraries with them. we try to package a library with the pattern. it would be worth some investigation if this is possible to compile a scss file along side these pattern definitions. see https://github.com/SU-HSDO/suhumsci/blob/develop/docroot/themes/humsci/su_humsci_theme/patterns/molecules/horizontal-card/horizontal-card.ui_patterns.yml#L44

Reply:
Yeah, there's a few trade-offs that we'd need to make in order to move in that direction. Our current style organization prioritizes "multiple-theme maintainability" over "UI patterns being portable." It provides some benefits, but I recognize that it isn't the same convention used in Jumpstart-UI and other projects.

You're right that it's worth investigation. I've created a JIRA card for us to explore what a transition would look like and what benefits that could provide.